datasets/lama/README.md

```diff
@@ -0,0 +1,314 @@
+* template_negated: Same as above, except the [Y] is not the object.
+* label: the label for the relationship/predicate
+* description': a description of the relationship/predicate
+* type: a type id for the relationship/predicate
+
+The conceptnet config has the following fields:
+* uuid: the id
+* sub: the subject
+* obj: the object to be predicted.
+* pred: the predicate/relationship
+* obj_label: the object label
+* masked_sentence: The masked sentence used to probe, with the object word replaced with [MASK]
+* negated: same as above, except [MASK] should predict something that is not the object word.
+
+
+The squad config has the following fields:
+* id: the id
+* sub_label: the subject label
+* obj_label: the object label that is being predicted
+* masked_sentence: The masked sentence used to probe, with the object word replaced with [MASK]
+* negated: same as above, except [MASK] should predict something that is not the object word.
+
+
+The google_re config has the following fields:
+
+* uuid: the id
+* pred: the predicate
+* sub: the subject
+* obj: the object
+* evidences: flattened json string that provides evidence for predicate. parse this json string to get more 'snippet' information.
+* judgments: data about judgments
+* sub_q: unknown
+* sub_label: label for the subject
+* sub_aliases: unknown
+* obj_w: unknown
+* obj_label: label for the object
+* obj_aliases: unknown
+* masked_sentence: The masked sentence used to probe, with the object word replaced with [MASK]
+* template: A pattern of text for extracting the relationship, object and subject of the form "[X] some text [Y]", where [X] and [Y] are the subject and object slots respectively. 
+* template_negated: Same as above, except the [Y] is not the object.
+
+
+### Data Splits
+
+There are no data splits.
+
+## Dataset Creation
+
+### Curation Rationale
+
+This dataset was gathered and created to probe what language models understand.
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+See the reaserch paper and website for more detail. The dataset was
+created gathered from various other datasets with cleanups for probing.
+
+
+#### Who are the source language producers?
+
+The LAMA authors and the original authors of the various configs.
+
+### Annotations
+
+#### Annotation process
+
+Human annotations under the original datasets (conceptnet), and various machine annotations.
+
+#### Who are the annotators?
+
+Human annotations and machine annotations.
+
+### Personal and Sensitive Information
+
+Unkown, but likely names of famous people.
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+The goal for the work is to probe the understanding of language models.
+
+### Discussion of Biases
+
+Since the data is from human annotators, there is likely to be baises. 

```

Comment:
```suggestion
Since the data is from human annotators, there is likely to be baises. 

[More Information Needed]
```

I'd add [More Information Needed] here

Reply:
one